model/host/host.go

```diff
@@ -2156,16 +2156,20 @@ func (h *Host) UpdateParentIDs(ctx context.Context) error {
 }
 
 // For spawn hosts that have never been set unexpirable, this will
-// prevent spawn hosts from being set further than 30 days.
+// prevent spawn hosts from being set further than 30 days and earlier
+// then the current time.
 // For unexpirable hosts, this will prevent them from being extended any further
-// than the new 30 day expiration (unless it is set to unexpirable again #loophole)
-func (h *Host) PastMaxExpiration(extension time.Duration) error {
+// than the new 30 day expiration (unless it is set to unexpirable again #loophole).
+func (h *Host) ValidExpirationExtension(extension time.Duration) error {
```

Comment:
nit: should change the function name to ValidateExpirationExtension since we're returning an error instead of a boolean